src/Components/CreateList/EditProdRow.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import './EditProdRow.css'
+import { AiFillDelete } from 'react-icons/ai'
+import { GiSaveArrow } from 'react-icons/gi'
+
+function EditProdRow({product, allProducts, index, editInputs, editProduct, handleEditSubmit, removeProduct}) {
+  return (
+    <div className="product-row-edit" key={index}>
+        <div className='outputs-edit'>
+          <div className='output-product'> {editInputs.product} </div>
+
+          {allProducts.filter(prod => prod.name === product.product)
```

Comment:
You only want the one item of `allProducts`, just accept it from the props and remove the `allProducts` & `product` props

Reply:
do you mean i should map + filter in GroceryForm.js and pass my item when i call <EditProdRow /> ?